src/webapp/src/client/app/common/directives/wall/comments/comment.repository.js

```diff
@@ -11,12 +11,14 @@
     ];
 
     function wallCommentRepository($resource, endPoint) {
+        var kudosUrl = endPoint + '/Kudos/';
```

Comment:
Now it's better, that front-end is not referring `kudosFactory`, but it's still not good, that it's using Kudos module back-end for that. Users autocomplete logic could be moved from kudos controller to users controller, as for example.